### DISCUSSION TITLE
如何修改定理类环境标题的字体与格式

### DISCUSSION THREAD

**Discussion by guoh064** [tuna/thuthesis#757] [Q&A] [answered]
如题，我希望将定理类环境标题的字体改为宋体/Times New Roman，并把格式改为加粗。
目前定理类环境标题与“证明：”使用的字体字号对应本科生《写作指南》中的四级节标题（黑体小四号、Arial 12pt）。
这似乎不被推荐使用，虽然 `ctex` 宏包也会给出以黑体为标题的命题。
中文还好，英文实在是丑……
一个想到的替代方案是采用类似 `\paragraph` 命令产生的加粗宋体/Times New Roman。

```
\documentclass[degree=bachelor]{thuthesis}
\usepackage{amsthm}

\begin{document}

\section{中文}
\begin{theorem}[一个简单的定理]
  任意自然数均可以写成四个自然数的平方和。
  \begin{proof}
    留做习题。
  \end{proof}
\end{theorem}

\paragraph{一个复杂的证明} 写不下了。

\thusetup{
  language = english,
}

\section{English}

\begin{theorem}[Easy Theorem]
  Any natural number is the sum of four square numbers.
  \begin{proof}
    Homework.
  \end{proof}
\end{theorem}

\paragraph{A Hard proof} Omitted.

\end{document}
```

![image](https://user-images.githubusercontent.com/50830808/161670874-d8f4df8d-7375-41e5-a20a-f97f82c26552.png)

**Comment by zepinglee** [comment, marked as answer]
将这里的 `\sffamily` 改为 `\bfseries` 既可以。
https://github.com/tuna/thuthesis/blob/d9869fd0f7f4b28c22354bcb4dd8dd17ca20a326/thuthesis.dtx#L6311

其实《写作指南》中并未规定 `\paragraph` 和定理的标题格式，可以自由发挥。模板中选择黑体作为定理标题的格式是因为 Windows 的粗宋体只能用伪粗，通常要尽量避免。英文的话其实使用粗体更合适。`\paragraph` 是保留了 LaTeX 原来的粗体格式。
